CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * [#8788](https://github.com/rubocop-hq/rubocop/issues/8788): Change `Style/Documentation` to not trigger offense with only macros. ([@tejasbubane][])
 
+### Bug fixes
+
+* [#9067](https://github.com/rubocop-hq/rubocop/pull/9067): Fix an incorrect auto-correct for `Lint::AmbiguousRegexpLiteral` when passing in a regexp to a method with no receiver. ([@amatsuda][])
```

Comment:
This is a nitpick. Can you move the Bug fixes section to above the Changes section? Alternatively, the changelog entry can be added in the new way.

> Added an entry (file) to the changelog folder named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See changelog entry format for details.

Reply:
Oops, sorry! Reordered!